src/MultiplicationTable.ts

```diff
@@ -1,5 +1,35 @@
 export class MultiplicationTable {
   public render(start: number, end: number): string{
-    return '1*1=1'
+    if (isValid(start, end) && isInRange(start, end)) {
+      var renderOutput = ''
+      for (let index = start; index <= end; index++) {
```

Comment:
Doesn't match with task diagram: 
missing: generateMultiplicationTable is embedded in render.